requirements.txt

```diff
@@ -1,4 +1,4 @@
-Django==1.4.18
+Django==1.7.7
```

Comment:
Did you also try upgrading all the other django dependencies?  Chances are that they needed to be updated to support django 1.7 as well